SignUpLogInPage/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>SignUp LogIn</title>
```

Comment:
O nome podia ser melhor mas vá aceito
```suggestion
  <title>Sign up and Log in</title>
```

ou 

```suggestion
  <title>SignUp LogIn Exercise</title>
```

Reply:
Devem pensar que eu tenho a vossa idade para andar a brincar aos PRs 😑